api/src/cli/index.test.ts

```diff
@@ -1,25 +1,30 @@
 import { Command } from 'commander';
 import { Extension, HookConfig } from '@directus/shared/types';
 import { createCli } from './index';
+import { test, expect, vi, beforeEach } from 'vitest';
 
-jest.mock('../../src/env', () => ({
-	...jest.requireActual('../../src/env').default,
-	EXTENSIONS_PATH: '',
-	SERVE_APP: false,
-	DB_CLIENT: 'pg',
-	DB_HOST: 'localhost',
-	DB_PORT: 5432,
-	DB_DATABASE: 'directus',
-	DB_USER: 'postgres',
-	DB_PASSWORD: 'psql1234',
-}));
+vi.mock('../../src/env', async () => {
+	const actual = await vi.importActual('../../src/env');
+
+	return {
+		EXTENSIONS_PATH: '',
+		SERVE_APP: false,
+		DB_CLIENT: 'pg',
+		DB_HOST: 'localhost',
+		DB_PORT: 5432,
+		DB_DATABASE: 'directus',
+		DB_USER: 'postgres',
+		DB_PASSWORD: 'psql1234',
+		...(actual as object),
+	};
+});
 
-jest.mock('@directus/shared/utils/node/get-extensions', () => ({
-	getPackageExtensions: jest.fn(() => Promise.resolve([])),
-	getLocalExtensions: jest.fn(() => Promise.resolve([customCliExtension])),
+vi.mock('@directus/shared/utils/node/get-extensions', () => ({
+	getPackageExtensions: vi.fn(() => Promise.resolve([])),
+	getLocalExtensions: vi.fn(() => Promise.resolve([customCliExtension])),
```

Comment:
This should probably look something like this:
```ts
vi.mock('@directus/shared/utils/node', async () => {
	const actual = await vi.importActual('@directus/shared/utils/node');

	const customCliExtension: Extension = {
		path: `/hooks/custom-cli`,
		name: 'custom-cli',
		type: 'hook',
		entrypoint: 'index.js',
		local: true,
	};

	return {
		...(actual as object),
		getPackageExtensions: vi.fn(() => Promise.resolve([])),
		getLocalExtensions: vi.fn(() => Promise.resolve([customCliExtension])),
	};
});
```

- `@directus/shared/utils/node` is exported in `@directus/shared` (sub-module `get-extensions` is not)
- However, with this I still get an error:
  
  ```console
  Caused by: Error: Error: Cannot find module /Users/pascal/Development/directus/api/Users/pascal/Development/directus/packages/shared/dist/esm/utils/node/index.js imported from file:///Users/pascal/Development/directus/api, file:///Users/pascal/Development/directus/, file:///Users/pascal/Development/directus/api/_index.js, file:///Users/pascal/Development/directus/node_modules
  ```
  
  Bug in `vitest`? Path looks wrong...